server/src/lsp4clj/coercer.clj

```diff
@@ -638,6 +637,11 @@
 (s/def :client-capabilities/signature-help (s/and ::debean
                                                   (s/keys :opt-un [:client-capabilities/signature-information])))
 
+(s/def :client-capabilities/resolve-support ::debean)
+
+(s/def :client-capabilities/completion-item (s/and ::legacy-debean
```

Comment:
We had some issues with the older debean, so I deprecated and created the new `::debean`,but I didn't change all places to use `::debean` to avoid lots of possible issues.

> And why do we use j/from-java, instead of j/from-java-deep

I don't know, didn't know that exists

Reply:
I switched from `::legacy-debean` to `::debean` and it's working fine, so this is ready for a final review. I played with `from-java-deep` a little. It's probably the right tool to use, but I couldn't get it to work... it was having some problems with reflection on JsonObject or JsonElement. But it's probably worth getting working—it'd simplify a lot of input coercion.